docs/index.md

```diff
@@ -3,44 +3,37 @@ title: Home
 nav_order: 1
 ---
 
-# Welcome
+# Welcome to the Santa documentation
 
-Santa is a binary authorization system for macOS. Here you will find the
-documentation for understanding how Santa works, how to deploy it and how to
-contribute.
+Santa is a binary authorization system for macOS. It consists of a system extension that monitors for executions using a set of rules stored in a local database, a GUI agent that notifies the user in case of a block decision, a `santasyncservice` binary responsible for syncing the server and database, and a command-line utility for managing the system.
```

Comment:
s/a `santasyncservice` binary/a sync daemon and s/the server and database/the database and a server/